docs/Lexicon.rst

```diff
@@ -291,6 +291,19 @@ source code, tests, and commit messages. See also the `LLVM lexicon`_.
     overlays" feature, which is used to augment a module with additional
     functionality when it is imported alongside another module.
 
+  parent type
+    The type where a given declaration is nested in. For example::
```

Comment:
"The type in which a given declaration is nested"

Reply:
Thanks, fixed.